packages/core/src/sass/app/_common/components/notification-banner.scss

```diff
@@ -7,7 +7,7 @@
     box-shadow: 0 4px 6px 0 var(--shadow-menu), 0 4px 6px 0 var(--shadow-menu);
     width: 100%;
     max-width: 40rem;
-    height: 14.4rem;
+    min-height: 14.4rem;
```

Comment:
Changes in this file will affect DP2P notification as well. Was it added to QA checklist?

Reply:
I'm gonna add it to the QA-Checklist.